src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -2138,29 +2138,33 @@ private string[] GetNativeDigits()
 
             // if digits.Length < NumberFormatInfo.s_asciiDigits.Length means the native digits setting is messed up in the host machine.
             // Instead of throwing IndexOutOfRangeException that will be hard to diagnose after the fact, we'll fall back to use the ASCII digits instead.
-            if (digits.Length >= NumberFormatInfo.s_asciiDigits.Length)
+            if (digits.Length < NumberFormatInfo.s_asciiDigits.Length)
```

Comment:
I think I'm missing something obvious -- why is this `<` rather than `!=`

Reply:
`digits` can include null termination character at the end.